setup.py

```diff
@@ -130,11 +134,9 @@ def get_extras_require() -> Dict[str, List[str]]:
             "pandas",
             "plotly>=4.0.0",
             "pytest",
-            "scikit-learn>=0.19.0,<0.23.0",
             "scikit-optimize",
             "xgboost",
             "keras",
```

Comment:
Don't we need to exclude this for Python3.9 as well as tensorflow-datasets and keras?